tests/e2e/workloads/app/pgsql/test_pgsql.py

```diff
@@ -34,10 +34,10 @@ def test_sql_workload_simple(self, pgsql):
         This is a basic pgsql workload
         """
         # Deployment postgres
-        pgsql.setup_postgresql(replicas=1)
+        pgsql.setup_postgresql(replicas=3)
 
         # Create pgbench benchmark
-        pgsql.create_pgbench_benchmark(replicas=1, transactions=600)
+        pgsql.create_pgbench_benchmark(replicas=3)
```

Comment:
Did you test with replicas setting to 3, is it working? AFAIK, there were some issues with replica  >1

Reply:
yes.. It is working